hypervisor/include/arch/x86/guest/vlapic.h

```diff
@@ -240,8 +240,8 @@ int vlapic_set_local_intr(struct acrn_vm *vm, uint16_t vcpu_id_arg, uint32_t vec
  * @param[in] addr MSI address.
  * @param[in] msg  MSI data.
  *
- * @return 0 on success.
- * @return non-zero on error that addr is invalid.
+ * @retval 0 on success.
```

Comment:
```suggestion
 * @retval 0 on success, non-zero otherwise (\p addr is invalid)
```

Reply:
Combining different return values in a single `@retval` directive makes the return value list incomplete at first glance. How about replacing `non-zero` with `<0` (the return value is always negative on error actually) to make to more explicit?